src/train.cpp

```diff
@@ -0,0 +1,35 @@
+ #include "train.hpp"
+ #include <iostream>
+ #include <string>
+ 
+ 
+ Trainer::Trainer()
+ {
+ }
+ Trainer::~Trainer()
+ {
+ }
```

Comment:
default constructor and destructor not needed. Compiler will generate them itself.